src/app/manager/hospital-map/services/navigation.service.ts

```diff
@@ -223,4 +230,64 @@ export class NavigationService {
     return this.markedFloor;
   }
 
+  private writeDirections(){
+      this.textPath.reverse();
+      var endPoint = this.destination?.id as string;
+      var forward = true;
+      this.navigationTips.push("Go to floor: " + this.markedFloor?.id);
```

Comment:
Refactor into methods and split logic. This is unmaintanable

Reply:
Nije ovo moje Filipovo je msm za navigaciju

Reply:
Vi jedni drugima pravite probleme, sad ako se tebi approve Filip ce imati konflikt xd

Reply:
ustvari nije Filipovo, Jelenino je  ovo je za onaj text za navigaciju

Reply:
nema veze cije je, dogovorite se i resite to ovde

Reply:
Rijesili smo ovo je stari kod i ne treba da bude tu tako da je sve izbrisano